src/main/java/uk/gov/hmcts/ccd/domain/service/common/SecurityClassificationService.java

```diff
@@ -183,46 +185,14 @@ private void filterObject(SecurityClassification userClassification, Iterator<Ma
         }
     }
 
-    private boolean isNull(JsonNode... jsonNodes) {
-        return newArrayList(jsonNodes).stream().anyMatch(Objects::isNull);
-    }
-
-    private JsonNode getDataClassificationForData(JsonNode data, Iterator<JsonNode> dataIterator) {
-        Boolean found = false;
-        JsonNode dataClassificationValue = null;
-        while (dataIterator.hasNext() && !found) {
-            dataClassificationValue = dataIterator.next();
-            if (null != dataClassificationValue.get(ID)
-                    && dataClassificationValue.get(ID).equals(data.get(ID))) {
-                found = true;
-            }
-        }
-        return dataClassificationValue;
-    }
-
     private void filterSimpleField(SecurityClassification userClassification, Iterator iterator, JsonNode dataClassificationValue) {
         Optional<SecurityClassification> securityClassification = getSecurityClassification(dataClassificationValue);
         if (!securityClassification.isPresent() || !userClassification.higherOrEqualTo(securityClassification.get())) {
             iterator.remove();
         }
     }
 
-    private Optional<SecurityClassification> getSecurityClassification(JsonNode dataNode) {
-        if (dataNode == null || dataNode.isNull()) {
-            return Optional.empty();
-        }
-        SecurityClassification securityClassification;
-        try {
-            securityClassification = valueOf(dataNode.textValue());
-        }
-        catch (IllegalArgumentException e) {
-            LOG.error("Unable to parse security classification for {}", dataNode, e);
-            return Optional.empty();
-        }
-        return Optional.of(securityClassification);
-    }
-
-    private Predicate<CaseDetails> caseIfHasClassificationEqualOrLowerThan(Optional<SecurityClassification> userClassificationOpt) {
-        return cd -> userClassificationOpt.map(uc -> uc.higherOrEqualTo(cd.getSecurityClassification())).orElse(false);
+    private boolean isNull(JsonNode... jsonNodes) {
+        return newArrayList(jsonNodes).stream().anyMatch(Objects::isNull);
```

Comment:
is anyMatch correct here? or should be allMatch?

Reply:
Yeah should be allMatch.